stock_quant_manual_assign/wizard/assign_manual_quants.py

```diff
@@ -64,7 +64,7 @@ def assign_quants(self):
         if move.picking_type_id.auto_fill_qty_done:
             # Auto-fill all lines as done
             for ml in move.move_line_ids:
-                ml.qty_done = ml.product_qty
+                ml.qty_done = ml.reserved_uom_qty
```

Comment:
Same, to put in its own commit so we can backport it (even if the field has changed in 16.0).

Reply:
@sebalix This is not meant to be backported as just introduced in v16